addons/contacts/views/contact_views.xml

```diff
@@ -51,7 +51,7 @@
         name="Configuration"
         parent="menu_contacts"
         groups="base.group_system"
-        sequence="2"/>
+        sequence="10"/>
```

Comment:
Now that you change this, set it to a higher value, [like it happens in other apps](https://github.com/odoo/odoo/blob/f40203c0f3954d3c6be7d4234ee102cd903704b6/addons/account/views/account_menuitem.xml#L46)


```suggestion
        sequence="35"/>
```